tests/virt_autotest/libvirt_nated_virtual_network.pm

```diff
@@ -35,11 +35,15 @@ sub run_test {
     upload_logs "vnet_nated.xml";
     assert_script_run("rm -rf vnet_nated.xml");
 
-    my ($mac, $model, $affecter, $exclusive);
+    my ($mac, $model, $affecter, $exclusive, $skip_type);
     my $gate = '192.168.128.1';
     foreach my $guest (keys %virt_autotest::common::guests) {
         record_info "$guest", "NAT BASED NETWORK for $guest";
-        ensure_online $guest, skip_network => 1;
+        #Just only 15-SP5 PV guest system have a rebooting problem due to bsc#1206250
+        $skip_type = ($guest =~ m/sles-15-sp5-64-pv/i) ? 'skip_ping' : 'skip_network';
+        #Ensures the given guests is started and fixes some common network issues
+        ensure_online $guest, $skip_type => 1;
```

Comment:
What does ```$skip_type => 1``` mean here ?

Reply:
during calling `ensure_online()` with 15-SP5 PV guest,  `$skip_type => 1` mean that `skip_ping => 1` , as mentioned, just only skip ping and also extend timeout for detecting ssh port currently. or else,   `$skip_type => 1` mean that skip_network => 1, without any check point for each given guest system.